packages/paste-core/components/alert-dialog/src/AlertDialog.tsx

```diff
@@ -109,47 +141,50 @@ export const AlertDialog = React.forwardRef<HTMLDivElement, AlertDialogProps>(
     },
     ref,
   ) => {
-    const transitions = useTransition(isOpen, getAnimationStates());
+    const transitions = useTransition(isOpen, AnimationStates);
     const headingID = useUID();
     const bodyID = useUID();
 
-    return (
-      <>
-        {transitions(
```

Comment:
We shouldn't need a fragment wrapping this so I removed some of the code wrapping this.